components/basic/LoginDropdown.js

```diff
@@ -12,30 +15,23 @@ const DropdownCard = () => {
     setIsUserLoggedIn,
   } = useAuth();
   const dropdownRef = useRef(null);
-  const userData = JSON.parse(localStorage.getItem("userData"));
-  const mentorData = JSON.parse(localStorage.getItem("mentorData"));
-  const router = useRouter();
```

Comment:
why you removed this?

Reply:
ok at that time I guess it is not needed ...but now I add this code pls check...@anmode

Reply:
ok cool!